src/sass/modules/_m-schemaform.scss

```diff
@@ -9,6 +9,12 @@ legend {
   padding: 0 0 .5em 0;
 }
 
+// Override IE
+fieldset {
```

Comment:
This is causing other pages in the app to have much smaller input boxes.

Reply:
Good catch, width has been set to 100%.
Large screen
![bs_win10_ie_11 0 2](https://user-images.githubusercontent.com/16051172/36168164-fdbe845e-10ac-11e8-89e2-606f8620f05d.jpg)
Small screen
![bs_win10_ie_11 0 3](https://user-images.githubusercontent.com/16051172/36168178-07958e0a-10ad-11e8-84d1-82ccbb5e9015.jpg)